README.md

```diff
@@ -101,6 +101,7 @@ The rest of the iOS Engineers work in the following squads:
       <td>Chesley Stephens</td><td><a href='https://github.com/cstephens-babylon'>@cstephens-babylon</a></td><td></td></tr>
       <tr><td>Patrick Westmeyer</td><td><a href='https://github.com/bh-pwestmeyer'>@bh-pwestmeyer</a></td><td></td></tr>
       <tr><td>Sam Francis</td><td><a href='https://github.com/SamFrancis-Babylon'>@SamFrancis-Babylon</a></td><td></td></tr>
+      <tr><td>Tyler Hargett</td><td><a href='https://github.com/thargett-babylon'>@thargett-babylon</a></td><td></td></tr>
```

Comment:
There's something off with the column alignment:

![image](https://user-images.githubusercontent.com/4175766/64621328-66f3d600-d3dd-11e9-96a2-307c96097e4d.png)

You probably need to increase the `rowspan` at line 100 to 4.